README.md

```diff
@@ -1,3 +1,21 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+###
+#### A 1-2 summary or bullet points outlining your plan for check-ins throughout the duration of the project.
+   Every morning, we are doing a stand-up meeting before class to check in about the project, give feedbacks and update each other.
+   Meet Friday morning to retro 
+
+#### A 2-4 sentence summary of your plan for project organization and workflow. This can include bullet points. If you plan to use a project management tool, please include a link to your project board.
+   - inheritance for game, teams and game_teams class (children to StatTracker class)
+   - hightest total score, highest scoring visitor and highest scoring home probably share methods (modules)
+
+#### A 2-3 sentence summary describing the different approaches your group discussed for project organization and how you collectively made a decision on which to use.
+   driver-navigator duo
+
+#### A 2-3 sentence summary describing your approach to the code design.
+
```

Comment:
Looking at the logic in each method and asking ourselves how many things each method is doing